typings/plugins/ColorVisionPlugin.d.ts

```diff
@@ -1,15 +1,22 @@
 import { BasePlugin } from '..';
 
+export const COLOR_BLIND_TYPES: string[];
+
 type ColorVisionPluginOptions = {
-  colorSelect?:string,
+  colorSelects?:string | HTMLElement,
```

Comment:
Shouldn't this be an array? (or `NodeList`?)

Reply:
The `ColorVisionPluginOptions` is just for the constructor parameters.
All of the constructors accept a selector string or a single HTML element. It's turned into an array of elements during the construction.

Reply:
Got it!